### PR TITLE
change byzer package naming

### LINKED ISSUE #1751: byzer-lang 日志文件改名
before
mlsql_engine.log

---

after
byzer-lang.log

### PR DESCRIPTION
**Pull request by hellozepp**
# What changes were proposed in this pull request?
- [ ] Finshed changes describe

# How was this patch tested?
- [ ] Testing done

# Are there and DOC need to update?
- [ ] Doc is finished

# Spark Core Compatibility

### CODE REVIEW

File: docs/gitbook/v1/howtouse/k8s_deploy.md [hunk 1]
```diff
@@ -57,7 +57,7 @@
             --conf spark.kryoserializer.buffer.max=200m 
             --conf "\"spark.executor.extraJavaOptions=-XX:+UnlockExperimentalVMOptions -XX:+UseZGC -XX:+UseContainerSupport  -Dio.netty.tryReflectionSetAccessible=true\""  
             --conf "\"spark.driver.extraJavaOptions=-XX:+UnlockExperimentalVMOptions -XX:+UseZGC -XX:+UseContainerSupport  -Dio.netty.tryReflectionSetAccessible=true\"" 
-            local:///tmp/streamingpro-mlsql-spark_3.0_2.12-1.7.0-SNAPSHOT.jar
```
Comment (chncaesar) [inline docs/gitbook/v1/howtouse/k8s_deploy.md:60]: Please change byzer-lang version 1.7.0-SNAPSHOT to 2.2.2

File: external/mlsql-autosuggest/pom.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>streamingpro_2.12</artifactId>
         <groupId>tech.mlsql</groupId>
-        <version>2.2.2</version>
+        <version>2.2.3-SNAPSHOT</version>
```
Comment (chncaesar) [inline external/mlsql-autosuggest/pom.xml]: Next version should be 2.3.0-SNAPSHOT.

**Review by chncaesar — CHANGES_REQUESTED** [review]

File: streamingpro-assembly/pom.xml [hunk 3]
```diff
@@ -5,15 +5,15 @@
...
+        <version>2.2.3-SNAPSHOT</version>
         <relativePath>../pom.xml</relativePath>
     </parent>
 
     <artifactId>streamingpro-assembly</artifactId>
     <packaging>pom</packaging>
 
     <build>
-        <finalName>mlsql-engine_${spark.bigversion}-${project.version}</finalName>
+        <finalName>byzer-lang-${spark.version}-${project.version}</finalName>
```
Comment (chncaesar) [inline streamingpro-assembly/pom.xml:16]: To Change mlsql-engine main jar to byzer-lang , please follow this code in assembly.xml:
```
    <files>
    <file>
        <source>${project.parent.basedir}/streamingpro-mlsql/target/streamingpro-mlsql-spark_${spark.bigversion}_${scala.binary.version}-${project.version}.jar</source>
        <outputDirectory>main</outputDirectory>
        <destName>byzer-lang-${spark.bigversion}_${scala.binary.version}-${project.version}.jar</destName>
    </file>
    </files>
```

And remove moduleSets
```
<!--    <moduleSets>-->
<!--        <moduleSet>-->
<!--            &lt;!&ndash; Enable access to all projects in the current multimodule build! &ndash;&gt;-->
<!--            <useAllReactorProjects>true</useAllReactorProjects>-->

<!--            &lt;!&ndash; Now, select which projects to include in this module-set. &ndash;&gt;-->
<!--            <includes>-->
<!--                <include>tech.mlsql:streamingpro-mlsql-spark_${spark.bigversion}_${scala.binary.version}</include>-->
<!--            </includes>-->
<!--            <excludes>-->
<!--&lt;!&ndash;                <exclude>tech.mlsql:streamingpro-common-${spark.bigversion}_${scala.binary.version}</exclude>&ndash;&gt;-->
<!--&lt;!&ndash;                <exclude>tech.msql:streamingpro-dl4j</exclude>&ndash;&gt;-->
<!--            </excludes>-->
<!--            <binaries>-->
<!--                <outputDirectory>libs</outputDirectory>-->
<!--                <includeDependencies>false</includeDependencies>-->
<!--                <unpack>false</unpack>-->
<!--            </binaries>-->
<!--        </moduleSet>-->
<!--    </moduleSets>-->
```

**Review by chncaesar — APPROVED** [review]

**Comment by hellozepp** [comment]
erQClD7oRJ

**Pull request merged**